### ISSUE TITLE
big performance hit

### ISSUE THREAD

**Issue by jckl-jackal** [lauralex/dwm_lut#18]
i really like this tool but with it enabled i lose about 20fps on warzone 2...

**Comment by lauralex** [comment]
Yes, unfortunately there is about 5% performance loss for graphics-intensive applications. I think it's something related to the LUT shader. I'll have to dig a bit into it.

**Comment by KingCrazyC** [comment]
Did you ever figure it out?

**Comment by KingCrazyC** [comment]
:(
